packages/driver-bench/src/unitBench/list.bench.mts

```diff
@@ -0,0 +1,123 @@
+import chalk from 'chalk';
+// FIXME: if you want to run these you will need to manually specify the import path.
```

Comment:
Did you intentionally port this file?  

I'm guessing it hasn't been run in a long while, it still tests `denque`, which we no longer depend on.

Reply:
I did bring it over, but did not make it runnable, I don't mind removing it. 

I figured we may, in future, set up some more tooling to test the unit benchable stuff in the driver. Like maybe server selection as in just the parsing and filtering of servers